nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -439,6 +439,7 @@ in
         gnome-shell-extensions
         gnome-themes-extra
         pkgs.gnome-tour # GNOME Shell detects the .desktop file on first log-in.
+        pkgs.nixos-artwork.wallpapers.simple-blue
         pkgs.nixos-artwork.wallpapers.simple-dark-gray
         pkgs.nixos-artwork.wallpapers.simple-dark-gray-bottom
```

Comment:
Maybe we can remove `simple-blue` and `simple-dark-gray-bottom`, and then merge #169553 as is, allowing users to select between pure gray & dual background. I doubt the `simple-blue` and `simple-dark-gray-bottom` will be very popular and people can always just install them if they want to use them so there is no need to clutter the background list with them.

Reply:
that would work but having both at the same time defeats the point of the feature and also doesn't allow you to choose the blue wallpaper with dark theme

Reply:
Thinking about it more, we probably do not want to include `simple-dark-gray` either. With #169553, we will list the now default dual background (for better or the worse). But I do not see much point in including any other backgrounds, unless we want to include the rest of the nixos-artwork set. If people want any of those, they can always just install them.

So could you please limit this to just the `pkgs/data/misc/nixos-artwork/wallpapers.nix` change?

Reply:
sure

Reply:
nixos-artwork also isn't something that's well documented and it's not something that's easy to figure out so I think a module option or a bit of documentation should be added